cloud/google/configs/ct-mirror-izenpe1.sh

```diff
@@ -0,0 +1,9 @@
+export INSTANCE_TYPE="mirror"
+export PROJECT="ct-mirror-izenpe1"
+export REGION="us-central1"
+export CLUSTER="ct-mirror-izenpe1"
+export ZONES="a c f"
```

Comment:
what's your strategy for choosing zones here?  unique triples taken from a, b, c, f apparently?